BE/Data.Access.Layer/Helpers/GlobalExceptionHandler/AppException.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Access.Layer.Helpers.GlobalExceptionHandler
+{
+    internal class AppException : Exception
```

Comment:
Why do we have two identical classes? Could they be centralized in a meaningful way?

Reply:
It's  a base, I think they won't be same in future, but anyway I just made it like this to test if I can catch exception from business/data layer in api layer.